src/app/query/bitcoin/fees/fee-estimates.query.ts

```diff
@@ -21,9 +27,11 @@ export function useGetAllBitcoinFeeEstimatesQuery<
   T extends unknown = FetchAllBitcoinFeeEstimatesResp,
 >(options?: AppUseQueryConfig<FetchAllBitcoinFeeEstimatesResp, T>) {
   const client = useBitcoinClient();
+  const network = useCurrentNetwork();
+
   return useQuery({
     queryKey: ['average-bitcoin-fee-estimates'],
```

Comment:
This is effectively a different query so need to amend query key

Reply:
Thanks for catching, will fix.